src/main/java/org/generationcp/middleware/dao/dms/ExperimentPropertyDao.java

```diff
@@ -391,8 +391,8 @@
 				trial.setInstanceId((Integer) row[3]);
 				trial.setSiteName((String) row[4]);
 				trial.setLocationName((String) row[4]);
-				if (row[5] != null && NumberUtils.isNumber((String) row[5])) {
-					trial.setLocationId(Integer.valueOf((String) row[5]));
+				if (row[5] != null && NumberUtils.isNumber(row[5].toString())) {
```

Comment:
Hi @gelli-leafnode , Use String.valueOf to prevent NPE in toString() when object is null